google/cloud/completion_queue_test.cc

```diff
@@ -453,6 +459,8 @@ TEST(CompletionQueueTest, ShutdownWithReschedulingTimer) {
 
   cq.Shutdown();
   t.join();
+  // This is a "it did not crash nor hung" test, if we get here it was
```

Comment:
"did not crash or hang" perhaps?  Or "neither crashed nor hung"?

Reply:
Fixed.